src/shared/generator.c

```diff
@@ -167,7 +167,9 @@ int generator_write_timeouts(
         usec_t u;
         int r;
 
-        r = fstab_filter_options(opts, "comment=systemd.device-timeout\0" "x-systemd.device-timeout\0",
+        r = fstab_filter_options(opts,
+                                 "comment=systemd.device-timeout\0" "x-systemd.device-timeout\0"
+                                 "comment=systemd.mount-timeout\0" "x-systemd.mount-timeout\0",
```

Comment:
Maybe I'm missing something, but `JobTimeoutSec` determines how long we'll wait for the device to show up. `mount-timeout` is about how long we wait after the device has shown up. I don't see how the latter translates into the firs.t